Sources/App/Services/TranslocService.swift

```diff
@@ -13,10 +13,15 @@ struct TranslocService: Service {
     /// Fetches the next arrival estimate for the given stop
     ///
     /// - Parameter stop: The desired `stop_id`
+    /// - Parameter route: The desired `route_id`
     /// - Returns: `TranslocData` which also contains the `name` of the stop, if available
     /// - Throws: Decoding errors
-    func time(for stop: String, agency: String = "32") throws -> Future<TranslocData> {
-        let url = baseUrl(for: .estimates, agency: agency) + "&stops=" + stop
+    func time(for stop: String, agency: String = "32", route: String) throws -> Future<TranslocData> {
+        
+        ///Add another pararmeter here--
+        let url = baseUrl(for: .estimates, agency: agency) + "&stops=" + stop + "&routes=" + route
```

Comment:
Now that `route` is an optional you'll need to change this to accommodate. Something like this should work: 

```swift
let url = baseUrl(for: .estimates, agency: agency) + "&stops=" + stop

if let routes = route {
    url += "&routes=" + routes
}
```